src/standardMethods.ts

```diff
@@ -306,19 +336,25 @@ export function generateRetrieveLatestEndpoint<RequestParams, ResponseType>(
 export function generateInsertEndpoint<RequestParams>(
   axiosInstance: AxiosInstance,
   resourcePath: string,
-  metadataMap: MetadataMap
+  metadataMap: MetadataMap,
+  chunkFunction?: (items: RequestParams[]) => RequestParams[][]
 ) {
   return async function insert(items: RequestParams[]): Promise<{}> {
-    const response = await rawRequest<ItemsResponse<{}>>(
-      axiosInstance,
-      {
-        method: 'post',
-        url: resourcePath,
-        data: { items },
-      },
+    let chunks: RequestParams[][] = [[]];
```

Comment:
I see a lot of duplicate code / logic for all the chunking across functions. Anyway we can refactor this?